databricks/koalas/plot.py

```diff
@@ -509,6 +509,17 @@ def _make_plot(self):
         super(KoalasBarhPlot, self)._make_plot()
 
 
+class KoalasScatterPlot(ScatterPlot, TopNPlot):
+    max_rows = 1000
```

Comment:
I think we can remove this.

Reply:
ahh, yeah, removed. and added pics in description, i thought i did it yesterday. my bad